app/services/hyrax/analytics/google.rb

```diff
@@ -48,6 +56,12 @@ def valid?
           REQUIRED_KEYS.each do |key|
             class_eval %{ def #{key};  @config.fetch('#{key}'); end }
           end
+
+          # This method allows setting the analytics id in the initializer
+          # which is deprecated
```

Comment:
Should this method have a deprecation warning as well?  I've seen `@deprecation` yard doc tags used elsewhere in Hyrax so maybe this method should at least have that.

Reply:
great point. updated yarn tag and what one should do instead.